themes/root/templates/RecordDriver/EDS/export-ris.phtml

```diff
@@ -0,0 +1,108 @@
+<?
+$items = $this->driver->getItems();
```

Comment:
I'm definitely in favor of making the EDS driver more uniform with the others, if this is technically feasible without significant data loss, duplication or misrepresentation. If I'm remembering correctly, the current code at least partially depends on rendering arbitrary key/value pairs provided by the API, as opposed to using the standard direct metadata access that most other VuFind backends use. If we could add more specific data accessors without also losing access to the EDS-specific data that falls outside of the standard VuFind mold, that would be great. Might it be feasible, for example, to maintain within a record driver a list of all of the fields that are specifically mapped, so that we could then continue to display any non-mapped fields using the current key/value mechanism? That might allow us to gain more granular control without losing any parts of our current displays. (Sorry if I'm misremembering and these comments are not actually relevant).

Regarding the author information, when we revised author indexing in 3.0, the record driver representation of authors should have become more flexible. There used to be the need to identify a single primary author. Now, in the new model, if we can't figure out the author roles, we can just treat the record as having multiple primary authors, which should be simpler and less opinionated. I believe that I made some adjustments to the EDS driver when this happened to try to make it work cleanly with the new system, but since I'm not an expert on this system, maybe it can be done better. I'd be happy to discuss that specific detail in greater depth if that would be helpful.